dc-cudami-editor/src/components/PagedIdentifiableList.jsx

```diff
@@ -0,0 +1,266 @@
+      debug,
+      mockApi,
+      parentType,
+      t,
+      type,
+      uiLocale,
+    } = this.props
+    return (
+      <ApiContext.Provider value={{apiContextPath, mockApi}}>
+        <Row>
+          <Col>
+            <h2>{t(`${type}s`)}</h2>
+          </Col>
+          <Col className="text-right">
+            <Button
+              onClick={() => this.toggleModal('addAttachedIdentifiables')}
+            >
+              {t('add')}
+            </Button>
+          </Col>
+        </Row>
+        <ListGroup className="identifiable-list">
+          <ListGroupItem className="pb-0 pt-0">
+            <Row className="text-center">
+              <Col className="border-right pb-2 pt-2 text-right" md="1">
+                <FaHashtag />
+              </Col>
+              <Col className="border-right pb-2 pt-2" md="1">
+                <FaImage />
+              </Col>
+              <Col className="border-right pb-2 pt-2" md="7">
+                {t('label')}
+              </Col>
+              <Col className="border-right pb-2 pt-2" md="2">
+                {t('lastModified')}
+              </Col>
+              <Col className="pb-2 pt-2" md="1">
+                {t('actions')}
+              </Col>
+            </Row>
+          </ListGroupItem>
+          {this.state.identifiables.map((identifiable, index) => (
+            <IdentifiableListItem
+              apiContextPath={apiContextPath}
+              index={index + 1 + this.state.pageNumber * this.pageSize}
+              key={index}
+              label={
+                identifiable.label[this.state.defaultLanguage] ??
+                Object.values(identifiable.label)[0]
+              }
+              lastModified={identifiable.lastModified}
+              onRemove={() => {
+                this.toggleModal('removeAttachedIdentifiable')
+                this.setState({removeIndex: index})
+              }}
+              previewImage={identifiable.previewImage}
+              previewImageRenderingHints={
+                identifiable.previewImageRenderingHints
+              }
+              type={type}
+              uiLocale={uiLocale}
+              uuid={identifiable.uuid}
+            />
+          ))}
+        </ListGroup>
+        {this.state.identifiables.length > 0 && (
+          <ReactPaginate
+            activeClassName="active"
+            breakClassName="page-item"
+            breakLabel="&hellip;"
+            breakLinkClassName="page-link"
+            containerClassName="pagination mt-2"
+            disabledClassName="disabled"
+            forcePage={this.state.pageNumber}
+            marginPagesDisplayed={1}
+            nextClassName="page-item"
+            nextLabel="&raquo;"
+            nextLinkClassName="page-link"
+            onPageChange={this.updatePage}
+            pageClassName="page-item"
+            pageCount={this.state.numberOfPages}
+            pageLinkClassName="page-link"
+            pageRangeDisplayed={5}
```

Comment:
Ist das nicht `this.pageSize` 🤔

Reply:
Nein, das definiert die Anzahl der Seiten, die in der Pagnierung anzeigt werden:
![screenshot-localhost_3000-2020 08 13-11_09_11](https://user-images.githubusercontent.com/19190936/90116394-b6612580-dd55-11ea-9b6c-f9c294ed937e.png)

Reply:
Ah, verstanden :)